po/fr.po

```diff
@@ -325,7 +320,7 @@ msgstr "trop court"
 
 #, c-format
 msgid "Bad password: %s.  "
-msgstr "Mot de passe erroné : %s."
+msgstr "Mot de passe erroné : %s. "
```

Comment:
A minor comment here:

The correct inter-sentence space ammount is two.

<https://git.kernel.org/pub/scm/docs/man-pages/man-pages.git/tree/CONTRIBUTING.d/patches/description#n44>
<https://web.archive.org/web/20171217060354/http://www.heracliteanriver.com/?p=324>

Anyway, since you're not the author, let's take the patch as is.